e2e/cypress/support/api/oidc-settings.ts

```diff
@@ -1,30 +1,32 @@
-import { apiCallProperties } from './apiauth';
-import { ensureSomethingIsSet } from './ensure';
+import { ensureSetting } from './ensure';
+import { API } from './types';
 
 export function ensureOIDCSettingsSet(
-  api: apiCallProperties,
-  accessTokenLifetime,
-  idTokenLifetime,
-  refreshTokenExpiration,
+  api: API,
+  accessTokenLifetime: number,
+  idTokenLifetime: number,
+  refreshTokenExpiration: number,
   refreshTokenIdleExpiration: number,
 ): Cypress.Chainable<number> {
-  return ensureSomethingIsSet(
+  return ensureSetting(
     api,
     `${api.adminBaseURL}settings/oidc`,
-    (settings: any) => {
-      let entity = null;
+    (body: any) => {
+      const result = {
+        entity: body.settings,
+        sequence: body.settings?.details?.sequence,
+        id: body.settings.id,
+      };
+
       if (
-        settings.settings?.accessTokenLifetime === hoursToDuration(accessTokenLifetime) &&
-        settings.settings?.idTokenLifetime === hoursToDuration(idTokenLifetime) &&
-        settings.settings?.refreshTokenExpiration === daysToDuration(refreshTokenExpiration) &&
-        settings.settings?.refreshTokenIdleExpiration === daysToDuration(refreshTokenIdleExpiration)
+        body.settings?.accessTokenLifetime != hoursToDuration(accessTokenLifetime) ||
```

Comment:
```suggestion
        body.settings?.accessTokenLifetime !== hoursToDuration(accessTokenLifetime) ||
```
use type checking wherever possible